### ISSUE TITLE
No description shows `undefined` in note for translator

### ISSUE THREAD

**Issue by Guibod** [evenchange4/react-intl-po#58]
If you don't have a message description, you see `undefined` in `#.` comments for `msgid`.

```
#: messages/_default.json
#. [Activities.filter.irrelevant.only] - undefined
msgid "Activities.filter.irrelevant.only"
msgstr "Afficher les mentions non-pertinentes"
```

**Comment by Guibod** [comment]
This behavior is fixed in this PR:
https://github.com/evenchange4/react-intl-po/pull/60

**Comment by evenchange4** [comment]
Thank you! It should  be fixed via #65 with your #60 changes.
Close it. Please feel free to reopen this issue if the problem still occurs with a newer version.
